api/stats.js

```diff
@@ -81,28 +82,37 @@ export class Stats {
                     // we couldn't parse
                     break;
                 }
-                stats[username] = stats[username] || this._defaultStats();
+                stats[username] = stats[username] || Stats._defaultStats();
                 stats[username].posts += Number(posts);
                 stats[username].words += Number(words);
                 stats[username].letters += Number(letters);
             }
         }
-
+        logger.debug("done getting weekly stats from channels");
         return stats;
     };
 
     _computeDailyStats = async () => {
         const dominationGuild = this.client.guilds.resolve(config.GUILD_ID);
         const channels = await dominationGuild.channels.fetch();
-        const stats = await this._initStats(dominationGuild);
+        const stats = await Stats._initStats(dominationGuild);
+
+        logger.debug("getting daily stats from channels");
```

Comment:
do we really need all this logging?

I wonder if we could just log the stack trace when everything breaks, instead of adding all this logging that we might never use.

eg: Do we have a real use case for this line?

Reply:
it's a strange one, I agree. By looking at the exceptions is hard to understand what we're doing in the app, perhaps we could focus on printing some kind of state that has useful meaning, and not generic messages for "I'm here" sort of logging